cli/cmd/dir.go

```diff
@@ -212,6 +217,7 @@ func init() {
 	cmdDir.Flags().BoolP("insecuressl", "k", false, "Skip SSL certificate verification")
 	cmdDir.Flags().BoolP("addslash", "f", false, "Apped / to each request")
 	cmdDir.Flags().BoolP("wildcard", "", false, "Force continued operation when wildcard found")
+	cmdDir.Flags().BoolP("head", "H", false, "Use HEAD requests if possible")
```

Comment:
Given the general use case, it might be worth defaulting to `HEAD` over `GET` and falling back if it doesn't work (or if the user asks for the length of the response). But given that we have the need/desire to handle other verbs, we might need to generalise this.

Reply:
:+1: for utilizing other verbs. It appears `-m` is available in `dir` mode, perhaps usage could look something like:

```
-m, --method               HTTP method/verb to use for request (GET, HEAD, etc.)
```

Should we also consider a flag that persists using the specified HTTP verb regardless of the web server response?